src/services/pegin-status/pegin-status.service.ts

```diff
@@ -110,10 +110,12 @@ export class PeginStatusService {
           this.logger.debug(errorMessage);
           this.status = Status.ERROR_NOT_A_PEGIN;
         } else {
+
           const federationAddress = await this.bridgeService.getFederationAddress();
```

Comment:
federationAddress is unnecessary in this context, please remove it.

Reply:
is used just after! in line 125. but i agree i can move it from line 114 to line 122